docs/hubs-cloud-aws-troubleshooting.md

```diff
@@ -0,0 +1,74 @@
+---
+id: hubs-cloud-aws-troubleshooting
+title: AWS Troubleshooting
+sidebar_label: AWS Troubleshooting
+---
+
+## You're in the "AWS Sandbox" and people don't receive "magic link" emails
+
+Your stack AWS email service is limited. The Hubs Admin console will also say you're in the "AWS Sandbox."
+
+#### Solution options:
+
+1.  **Send request to AWS for a limit increase**
+
+    - Use this process if you have 24 hours to wait for AWS technical support to grant your email service limit increase
+    - Go to [AWS create case](https://console.aws.amazon.com/support/home?#/case/create?issueType=technical)
+    - Select "Service limit increase"
+    - Fill in your email and copy & paste this message below:
+
+    ```
+    Hi there, we are deploying a product called Hubs Cloud to our AWS infrastructure. This product uses "magic links" sent via email to log in users. The product does not support passwords, only email links to log in. As such, would like to use SES to send these emails. There are no other emails sent by the product other than these automated sign-in emails, sent at the time a user requests to log in. We do not send any unsolicited emails or other content-oriented emails, only sign in link emails. We do not store the email addresses of visitors. The emails are securely transmitted to our server at log-in time to send the email link.
+
+    We expect to have an influx of users over the coming weeks using Hubs Cloud. As such, we need to be able to send these log in emails for them to use the site.
+
+    We've set up bounce and complaints to forward to an administrative email address.
+    Thank you!
+    ```
+
+    - Submit form
+    - Email will arrive granting your service limit increase request
+
+2.  **Setup SMTP via SendGrid**
+    - Use this process if you need an immediate fix
+    - Sign up on [SendGrid](https://sendgrid.com/)
+    - Create a new API Key
+    - Grant the API key email send ability
+    - Update settings in Hubs Cloud Admin console
+      1.  Server settings > SMTP
+      2.  Enter API key as "Username" _and_ "Password"
+      3.  Enter "smtp.sendgrid.net" as "Host"
+      4.  Enter "587" as "Host" first - if emails are blocked by spam change to "2525"
+
+## "ExternalZoneSSLCertLocalIfEast" error or timeout
+
+You have an issue with SSL certificate verification
+
+#### Potential solutions:
+
+1. **Check your domains**
+   - Are your domain names typed correctly?
+   - Check that you registered the domains on AWS Route 53
```

Comment:
Many people have been confused by the process of setting up a domain on route 53 and properly registering nameservers - we probably should write a detailed walkthrough with screenshots. I suspect this accounts for 90% of these failures at this point now that I've known to ask about it.

Reply:
Following up on this!